src/playlist-maker.js

```diff
@@ -3,6 +3,16 @@ var setupAutoadvance = require('./autoadvance.js');
 var isArray = Array.isArray || function(array) {
   return Object.prototype.toString.call(array) === '[object Array]';
 };
+var isInSources = function(arr, src) {
+  var i = 0;
+  for (; i < arr.length; i++) {
+    if (arr[i] === src || arr.src === src) {
```

Comment:
It should take the playlist object and tell us whether a given source (`src`) is in there.
So, something like:
```js
[{
  sources: ['http://example.com/video.mp4', 'http://example.com/video.webm']
}, {
  sources: [{
    src: 'http://example.com/another-video.mp4',
    type: 'video/mp4'
  }, {
    src: 'http://example.com/another-video.webm',
    type: 'video/webm'
  }
}]
```
and then given `'http://example.com/video.webm'` or `'http://example.com/last-video.mp4'`, we should get a `true` and a `false`.